Firestore/Example/Tests/Remote/FSTSerializerBetaTests.mm

```diff
@@ -439,23 +439,26 @@ - (void)assertRoundTripForMutation:(FSTMutation *)mutation proto:(GCFSWrite *)pr
 }
 
 - (void)testDecodesMutationResult {
+  SnapshotVersion commitVersion{Timestamp{0, 3000}};
```

Comment:
Optional: You could just specify this inline with testutil::Version (as `Version(3000)`).

Reply:
I switched to the `testutil`, but I don't want to inline it and I'm un-inlining the other so that it's clear that the versions actually match.